### ISSUE TITLE
After trained model, the original classification accuracy decreases

### ISSUE THREAD

**Issue by red-gezi** [longzw1997/Open-GroundingDino#51]
你好，我想要在原模型可识别类别基础上新增我想要的类别
比如这个图，我想新增角落文字的识别，使用”OSD“这个标签
Hello, I want to add a new category to the original model based on the recognized categories.
For example, in this image, I want to add the recognition of corner text using the label "OSD"
![iwEcAqNwbmcDAQTRBPMF0QMgBrCPSOYzBEbSYQWGPffK_TEAB9IUpLwSCAAJomltCgAL0gAdIdc png_720x720q90](https://github.com/longzw1997/Open-GroundingDino/assets/18701431/01810f66-de0f-4027-9429-6321ab7030a7)
然后我在训练后发现，虽然能有效识别文字目标了，但是car无法识别了，person识别率也大幅度下降了
Then I found after training that although I could effectively recognize text targets, the car could not be recognized, and the person recognition rate also dropped significantly.
![iwEcAqNwbmcDAQTRBO0F0QMsBrAJrD3sv7N3FQWGPffK_TEBB9IUpLwSCAAJomltCgAL0gAc8HU png_720x720q90](https://github.com/longzw1997/Open-GroundingDino/assets/18701431/ab09bab1-148d-4dbc-a0c9-3f935d7844f4)
以下是我用自己的脚本按照项目指引生成的训练集和验证集
The following are the training and validation sets generated by my own script according to the project guidelines
训练集格式
Training set format
![iwEdAqNwbmcDAQTRA8cF0QFkBrC5JjmYoW_baAWGQCDFyugAB9IUpLwSCAAJomltCgAL0V_x png_720x720q90](https://github.com/longzw1997/Open-GroundingDino/assets/18701431/ad77fdf9-d6d4-4529-8a82-4b394bdc1b38)
验证集格式
Validation set format
![iwEcAqNwbmcDAQTRA20F0QEWBrCjRMhEyQui3AWGQCDFyugCB9IUpLwSCAAJomltCgAL0T-x png_720x720q90](https://github.com/longzw1997/Open-GroundingDino/assets/18701431/5d5bd1b2-50fd-4d9d-b1bd-ec70972bacd2)
新特征列表
Validation set format
![iwEcAqNwbmcDAQTRAS4F0QDOBrBRG_Q7sbzfBgWGQCDFyugBB9IUpLwSCAAJomltCgAL0R1N png_720x720q90](https://github.com/longzw1997/Open-GroundingDino/assets/18701431/1d5fdb63-5070-4952-ab26-7035f7c711c9)
请问问题可能出在哪里
Please tell me where the problem may be
